pkg/discovery/const.go

```diff
@@ -4,6 +4,7 @@ const (
 	ClusterIdLabel      = "liqo.io/cluster-id"
 	AuthTokenLabel      = "liqo.io/auth-token"
 	RemoteIdentityLabel = "liqo.io/remote-identity"
+	DiscoveryTypeLabel  = "liqo.io/discovery-type"
```

Comment:
what do you think about adding the `discovery` prefix to all these labels?

Reply:
Do you mean something like "discovery.liqo.io/..."?

Reply:
yep

Reply:
if you think it is appropriate for these labels of course